src/main/resources/de/thetaphi/forbiddenapis/signatures/jdk-deprecated-17.txt

```diff
@@ -0,0 +1,632 @@
+# This file contains API signatures extracted from the rt.jar / jimage file shipped with the class library of Oracle's Java Runtime Environment.
+# It is provided here for reference, but can easily regenerated by executing from the source folder of forbidden-apis:
+# $ ant generate-deprecated
+
+# This file contains all public, deprecated API signatures in Java version 17 (extracted from build 17-ea).
```

Comment:
Downloaded the [OpenJDK early-access build](https://jdk.java.net/17/) today. Version details:
```
$ java -version
openjdk version "17-ea" 2021-09-14
OpenJDK Runtime Environment (build 17-ea+31-2664)
OpenJDK 64-Bit Server VM (build 17-ea+31-2664, mixed mode, sharing)
```